README.md

```diff
@@ -420,9 +420,11 @@ layers:
     collectors:
       - type: className
```

Comment:
Something unrelated that I just noticed: this could simply be the class FQN, and then customising all this stuff would be a breeze.

Reply:
opened an issue https://github.com/sensiolabs-de/deptrac/issues/164